README.md

```diff
@@ -1,21 +1,24 @@
 # Adyen [Magento 2 Plugin](https://docs.adyen.com/plugins/magento-2) integration demo
 
-In this demo, you can spin up a Magento instance and install the Adyen plugin to see how an integration works. It gives your shoppers the option to pay with their preferred payment method in a seamless checkout experience. You can try this demo both online or on your computer.
+In this demo, you can spin up a Magento instance and install the [Adyen Payment plugin](https://marketplace.magento.com/adyen-module-payment.html) to see how it works. You can try this demo online in one click using Gitpod (no local installation/dependencies) or locally on your computer using Docker.
 
 ## Running online with [Gitpod](https://gitpod.io/)
 
 1. Open your [Adyen Test Account](https://ca-test.adyen.com/ca/ca/overview/default.shtml) and create a set of [API keys](https://docs.adyen.com/user-management/how-to-get-the-api-key).
-2. > __Note__ _Step 2 is optional, you can skip to step 3 if you would like to manually input your API Keys in the Magento Admin Dashboard_ 
- - _Go to [gitpod account variables](https://gitpod.io/variables)_
- - _Set the `ADYEN_API_KEY`, `ADYEN_CLIENT_KEY` and `ADYEN_MERCHANT_ACCOUNT` variables (Scope: `adyen-examples/*`)._
+2. 
+ - Go to [gitpod account variables](https://gitpod.io/variables)
+ - Set the `ADYEN_API_KEY`, `ADYEN_CLIENT_KEY` and `ADYEN_MERCHANT_ACCOUNT` variables (Scope: `adyen-examples/*`).
+ - Create your login variables `ADMIN_USERNAME` and `ADMIN_PASSWORD` to access the magento admin dashboard. (Scope: `adyen-examples/*`).
```

Comment:
Create login variables might not be clear to the developer/user. Suggestion (feel free to change to something similar) : Set `ADMIN_USERNAME` and `ADMIN_PASSWORD` in the gitpod account variables as well. These will used as your login credentials in the magento admin dashboard.